proposals/001-kroxylicious-operator-api-v1alpha.md

```diff
@@ -364,9 +356,8 @@ spec:
   ingress:
   - name: myclusterip
 
-  targetCluster:
-    clusterRef:
-      name: mytargetcluster
+  targetKafkaServiceRef:
```

Comment:
would `proxiedKafkaServiceRef` be more easily understood by users?

Reply:
The comment above is `# Points to the cluster being proxied`. It's interesting that such an explanation is necessary, and that that's the language we use there.

Reply:
I dunno, I can see arguments both ways really. 

For me `target` encapsulates what it actually and I think fairly obvious what it means. My other hesitation is `proxiedKafkaService` is the result of applying the CRs so feels like a more natural name for a thing describing how to connect to Kafka via the proxy. 

But I can certainly see `proxiedKafkaServiceRef` as being easier to understand when just reading a CR without any context. 

That said the terminology we use in the proxy is `upstream` so to be consistent it should be `upstreamKafkaServerRef`

Reply:
I'm fine with merging as-is.

> That said the terminology we use in the proxy is upstream so to be consistent it should be upstreamKafkaServerRef

Would be more compelling if we actually _liked_ that terminology. But we've questioned it in the past, so...
Also that terminology is usually used in a slightly different context: where we have two connections and want to distinguish them with a single qualifier and it's desirable for the one qualifier to be "the opposite" of the other. In contexts like this one we don't use the term "downstream cluster".

Reply:
issue to carry on https://github.com/kroxylicious/design/issues/63